lib/config-generator.js

```diff
@@ -320,20 +320,26 @@ class ConfigGenerator {
          *  * https://github.com/webpack/webpack.js.org/issues/652#issuecomment-273324529
          *  * https://webpack.js.org/guides/caching/#deterministic-hashes
          */
-        // when not using versioning, none of this is important
-        if (this.webpackConfig.useVersioning) {
-            let moduleNamePlugin;
-            if (this.webpackConfig.isProduction()) {
+        let moduleNamePlugin;
+        if (this.webpackConfig.isProduction()) {
+            // only enable if versioning is on, as it makes the assets *slightly* larger
```

Comment:
@stof Unless you enable versioning, your output filenames will *always* be static and not changing (e.g. `main.js`). This just controls the internal names used for the modules. Or am I missing a detail? Basically, I don't think the module naming scheme should be important externally.